### PR TITLE
added documentation for using Sulu in combination with Heroku

### PR DESCRIPTION
**Pull request by danrot**

### CODE REVIEW

File: cookbook/running-sulu-on-heroku.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Running Sulu on Heroku
+======================
+
+`Heroku`_ enables programmers to run their applications in the cloud, and makes
+it really easy to scale your application if it follows some rules. There is a 
+`Sulu cloud edition`_, which follows these rules.
+
+You find a "Deploy to Heroku" button on the ``README.md`` file in the
```
Comment (dantleech) [inline cookbook/running-sulu-on-heroku.rst]:
````
You will find .. in the ``README...`` file of the `Sulu cloud edition`_.
````

File: cookbook/running-sulu-on-heroku.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+repository of the previously mentioned `Sulu cloud edition`_. This button leads
+to a heroku page for deploying your very own Sulu installation.
```
Comment (dantleech) [inline cookbook/running-sulu-on-heroku.rst]: Big "H" I think. Heroku

File: cookbook/running-sulu-on-heroku.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Give the application a name and choose between ``Europe`` and ``United States``
+as a region, whatever is closer to your actual destination. This choice will
+also influence the performance of your website.
+
+Unless you really want to, you should leave the ``SYMFONY_ENV`` environment
+variable to ``prod``. Otherwise the buid process will fail, because heroku
+installs the dependencies with ``composer install --no-dev``. If you change the
+environment to ``dev`` or ``test`` Sulu wil try to load some Symfony bundles,
```
Comment (dantleech) [inline cookbook/running-sulu-on-heroku.rst]: `Sulu will`

**Comment by danrot** [comment]
@dantleech Fixed and changed the second last paragraph. Can you please merge if it is ok?

File: cookbook/running-sulu-on-heroku.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+`Heroku`_ enables programmers to run their applications in the cloud, and makes
+it really easy to scale your application if it follows some rules. There is a 
+`Sulu cloud edition`_, which follows these rules.
+
+You will find a "Deploy to Heroku" button in the ``README.md`` file of the
+repository of the previously mentioned `Sulu cloud edition`_. This button leads
+to a Heroku page for deploying your very own Sulu installation.
+
+Give the application a name and choose between ``Europe`` and ``United States``
```
Comment (dantleech) [inline cookbook/running-sulu-on-heroku.rst]: I would say "Europe" and "United States" should be in normal (double) quotes. Double back-ticks should be reserved for things that would be parsed by something other than a human.

File: cookbook/running-sulu-on-heroku.rst [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+also influence the performance of your website.
+
+Unless you really want to, you should leave the ``SYMFONY_ENV`` environment
+variable to ``prod``. Otherwise the buid process will fail, because Heroku
+installs the dependencies with ``composer install --no-dev``. If you change the
+environment to ``dev`` or ``test`` Sulu wil try to load some Symfony bundles,
+which are not installed, and therefore lead to an error during the build
+procedure.
+
+The other environment variable you have to set is the ``DOMAIN``. Set it to
```
Comment (dantleech) [inline cookbook/running-sulu-on-heroku.rst]:
````
is ``DOMAIN``
````

(no "the"), or:

````
You must also set the ``DOMAIN`` environment variable.
````

**Comment by danrot** [comment]
@dantleech fixed

**Pull request merged**